presentation/components/src/presentation-components/tree/controlled/PresentationTreeNodeRenderer.tsx

```diff
@@ -22,25 +24,44 @@ export function PresentationTreeRenderer(props: TreeRendererProps) {
   />;
 }
 
+interface FilterButtonProps {
+  onFilter?: (node: TreeModelNode) => void;
+}
+
+const FilterButton = (props: FilterButtonProps) => {
+  return (
+    <IconButton className="presentation-filter-action-button" styleType="borderless"
+      onClick={() => props.onFilter}>
+      <SvgFilter className="presentation-filter-icon"/>
```

Comment:
Does not look like `presentation-filter-icon` class provide any styling. Remove it?